### PR TITLE
Adds the apscheduler and some pre-configured reminders

### PR DESCRIPTION
**Pull request by DESm1th**
This should be merged in alongside the datman pull request.

### CODE REVIEW

**Comment by DESm1th** [comment]
Note to self: Add the new environment variables to the uwsgi config file after merging

File: dashboard/emails.py [hunk 1]
```diff
@@ -74,3 +74,22 @@ def account_rejection_email(user):
             "and unfortunately it has been rejected. For any questions you "\
             "may have please contact us at {}".format(DASH_SUPPORT)
     send_email(subject, body, recipient=user.email)
+
```
Comment (josephmje) [inline dashboard/emails.py]: I missed this previously but on line 11, `async` is a reserved keyword in Python 3.7. Should the function name be changed?

File: dashboard/models.py [hunk 2]
```diff
@@ -344,6 +346,7 @@ def add_timepoint(self, timepoint):
             e.message = "Failed to add timepoint {}. Reason: {}".format(
```
Comment (josephmje) [inline dashboard/models.py]: Line 340, `FlushError` function not imported.

File: dashboard/emails.py (hunk 1 above)
Comment (DESm1th) [reply dashboard/emails.py]: Good catch! Currently I don't think the dashboard is at all python3 compatible but I'll change it now so I don't shoot myself in the foot later.

File: dashboard/models.py (hunk 2 above)
Comment (DESm1th) [reply dashboard/models.py]: Ah! Whoops. Another good catch, thank you!

File: dashboard/models.py [hunk 3]
```diff
@@ -247,6 +247,7 @@ def __repr__(self):
     def __str__(self):
         return "{} {}".format(self.first_name, self.last_name)
```
Comment (josephmje) [inline dashboard/models.py]: in `add_redcap`, add 
```
if event_id:
    rc_record.event_id = event_id
```

**Pull request merged**